lib/common/init.ts

```diff
@@ -1,29 +1,35 @@
-'use strict'
+import * as timers from 'timers'
+import * as util from 'util'
 
-const timers = require('timers')
-const util = require('util')
+import { atomBindingSetup } from '@electron/internal/common/atom-binding-setup'
 
-process.atomBinding = require('@electron/internal/common/atom-binding-setup')(process.binding, process.type)
+process.atomBinding = atomBindingSetup(process.binding, process.type)
 
 // setImmediate and process.nextTick makes use of uv_check and uv_prepare to
 // run the callbacks, however since we only run uv loop on requests, the
 // callbacks wouldn't be called until something else activated the uv loop,
 // which would delay the callbacks for arbitrary long time. So we should
 // initiatively activate the uv loop once setImmediate and process.nextTick is
 // called.
-const wrapWithActivateUvLoop = function (func) {
+const wrapWithActivateUvLoop = function <T extends Function> (func: T): T {
   return wrap(func, function (func) {
-    return function () {
+    return function (this: any) {
       process.activateUvLoop()
       return func.apply(this, arguments)
-    }
+    } as any
```

Comment:
Can we do better than `any`? It seems like it'll at least be `(...args: Array<any>) => any`.